[Bug]: TUnitImplicitUsings / TUnitAssertionsImplicitUsings not respected in Directory.Build.props

### Description

All test projects are configured using a single property set at the folder level in Directory.Build.props
If I want to prepare migrating it to TUnit, I want both TUnit and NUnit to be present in the projects to run code fixers.
However, the migration workaround to remove explicit usings:
```xml
<PropertyGroup>
    <TUnitImplicitUsings>false</TUnitImplicitUsings>
    <TUnitAssertionsImplicitUsings>false</TUnitAssertionsImplicitUsings>
</PropertyGroup>
```
doesn't work if I try to apply it in Directory.Build.props.

### Expected Behavior

Projects build successfully, there are no namespace conflicts.

### Actual Behavior

error CS0104: 'Test' is an ambiguous reference between 'TUnit.Core.TestAttribute' and 'NUnit.Framework.TestAttribute'

### Steps to Reproduce

1. Create NUnit test project
2. Add TUnit to Directory.Build.props
3. Add disabling implicit usings to Directory.Build.props
4. Build

### TUnit Version

1.20.0

### .NET Version

.NET 10

### Operating System

Windows

### IDE / Test Runner

dotnet CLI (dotnet test / dotnet run)

### Error Output / Stack Trace

```shell
error CS0104: 'Test' is an ambiguous reference between 'TUnit.Core.TestAttribute' and 'NUnit.Framework.TestAttribute'
```

### Additional Context

Putting those properties directly into csproj files works.

### IDE-Specific Issue?

- [x] I've confirmed this issue occurs when running via `dotnet test` or `dotnet run`, not just in my IDE

--- a/TUnit.Assertions/TUnit.Assertions.props
+++ b/TUnit.Assertions/TUnit.Assertions.props
@@ -16,7 +16,7 @@
         </PropertyGroup>
     </Target>
 
-    <PropertyGroup Condition="'$(XunitPackageReferenceFound)' == 'false'">
+    <PropertyGroup Condition="'$(XunitPackageReferenceFound)' == 'false' And '$(TUnitAssertionsImplicitUsings)' == ''">
         <TUnitAssertionsImplicitUsings>true</TUnitAssertionsImplicitUsings>
     </PropertyGroup>
 

--- a/TUnit/TUnit.props
+++ b/TUnit/TUnit.props
@@ -13,8 +13,8 @@
     </PropertyGroup>
 
     <PropertyGroup>
-        <TUnitImplicitUsings>true</TUnitImplicitUsings>
-        <TUnitAssertionsImplicitUsings>true</TUnitAssertionsImplicitUsings>
+        <TUnitImplicitUsings Condition="'$(TUnitImplicitUsings)' == ''">true</TUnitImplicitUsings>
+        <TUnitAssertionsImplicitUsings Condition="'$(TUnitAssertionsImplicitUsings)' == ''">true</TUnitAssertionsImplicitUsings>
     </PropertyGroup>
 
     <ItemGroup>

